tests/constraint/test_SE2BoxConstraint.cpp

```diff
@@ -21,15 +21,15 @@ class SE2BoxConstraintTests : public ::testing::Test
 {
 protected:
 #ifndef NDEBUG
-  static constexpr std::size_t NUM_A_TARGETS{5};
   static constexpr std::size_t NUM_X_TARGETS{5};
   static constexpr std::size_t NUM_Y_TARGETS{5};
+  static constexpr std::size_t NUM_A_TARGETS{5};
   static constexpr std::size_t NUM_SAMPLES{100};
-  static constexpr double DISTANCE_THRESHOLD{0.9};
+  static constexpr double DISTANCE_THRESHOLD{1.5};
```

Comment:
I am actually not sure what this threshold should be. It's a cool math problem though, if anyone figures it out, please let me know! The problem is:

Assume a unit square is discretized at some resolution `r`. Given `n` samples, I must be able to say that every grid point has at least one sample amongst the `n` that is at least as close as `x`. Find an upper bound on `x` given `n` and `r`.

Reply:
Why does this threshold need to change?

Reply:
Sorry, I guess my question is really why do both `NUM_SAMPLES` and `DISTANCE_THRESHOLD` need to change? I'd rather keep `NUM_SAMPLES` the same and just increase the threshold. (Again, this change is just to ensure that non-debug builds are as fast as debug builds.)

Reply:
I increased the number just for it to be consistent with the non-debug style i.e number of samples in the grid = number of samples used for coverage tests. It shouldn't change the build speed much :)

The changes made in #418 for faster tests should hold.